internal/cmdget/cmdget.go

```diff
@@ -101,3 +103,12 @@ func (r *Runner) runE(c *cobra.Command, _ []string) error {
 
 	return nil
 }
+
+func (r *Runner) postRunE(_ *cobra.Command, _ []string) error {
+	const op errors.Op = "cmdget.postRunE"
```

Comment:
This seems like a pretty important part of kpt functionality, so I don't think we should put it as a step in the cobra command. We should do this as part of the `get.Command` functionality.
Also, this must also happen for update. We might be pulling in new resources from upstream that doesn't have the comment.

Reply:
Updated.